src/Components/Login/Login.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import LoginForm from './LoginForm/LoginForm'
+
+const Login = () => {
+    return <div>
+        {/* form of logging in */}
```

Comment:
The comment is not so useful cause role of <LoginForm> seems clear from naming

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/pages/login.jsx#L8-L15